### PR TITLE
Prevent crate move on mission LOG_Medical

### PR DESCRIPTION
**Pull request by Xyaren**
* Some refactoring for cleaner crate creation and deletion
* Prevent crates from beeing moved via ace (dragging/carrying)
* Prevent crates from beeing loaded into a vehicle via ACE_cargo
* Use checks to prevent mod errors

### CODE REVIEW

File: AntistasiOfficial.Altis/Missions/LOG_Medical.sqf [hunk 1]
```diff
@@ -69,30 +69,37 @@ _sboxempty = createVehicle [_tipoVeh, _poscrash, [], 0, "CAN_COLLIDE"];
...
+
+_crates = [];
+{
+	_x params ["_distance","_heading"];
+	
+	_currentCrate =  "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
+	_currentCrate setPos ([getPos _sboxempty, _distance, _heading] call BIS_Fnc_relPos);
+	_currentCrate setDir (getDir _sboxempty + (floor random 180));
+
+	if (isClass(configfile >> "CfgPatches" >> "ace_dragging")) then { // check if ace dragging module is active
```
Comment (IrLED) [inline AntistasiOfficial.Altis/Missions/LOG_Medical.sqf]: That check will work (until ACE changes its internals)

What about checking ACE modules with provided `ace_common_fnc_isModLoaded`
https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_isModLoaded.sqf

See https://github.com/A3Antistasi/antistasiofficial/blob/master/AntistasiOfficial.Altis/Functions/fn_detectACE.sqf

Comment (IrLED) [reply AntistasiOfficial.Altis/Missions/LOG_Medical.sqf]:
```
if(activeACE && {["ace_dragging"] call ace_common_fnc_isModLoaded}) then {
...
};
```

**Comment by IrLED** [comment]
@StefArma 
I don't know if those crates should be interacted or not.

**Comment by StefArma** [comment]
Thanks for the contribution! 
Yes, those crates shouldn't be moved around nor loaded into cargo.

File: AntistasiOfficial.Altis/Missions/LOG_Medical.sqf (hunk 1 above)
Comment (Xyaren) [reply AntistasiOfficial.Altis/Missions/LOG_Medical.sqf]: But you don't have the function if ace is not loaded !? So what's the point of it ?

**Comment by Xyaren** [comment]
Fixed ace-check in f9c9f87d49913130354e7d90bd35db1a32a90042.
Maybe extract the whole ace part to a function to be reusable ? 
There might be more objects that should not be ace loadable to vehicles for example.

**Pull request merged**